PiwikTracker/Context.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+
+public class Context {
```

Comment:
imho, having nested classes with only static members (all internally available) does not add any value.
We are using namespace to avoid clashes and `Device` and `Application` does not share any `fileprivate` code not visible to the outside.

I would just make a stand-alone `Device` struct (prefer struct over class, or make class `final`)